doc/releasing.md

```diff
@@ -48,6 +49,7 @@ Do the following when releasing:
         +silk-create-asset-group \
         --branch <branch>
      ```
+   - Create a new Snyk reference target for the branch. See [Checking Snyk](https://docs.google.com/document/d/1SEUAuF923dVPL_Oq4P-TNXEby8u8fVOCcfwYrtHA-YQ/edit).
```

Comment:
May want to reword/defer this instruction to create a new Silk Asset Group instead.